conf/swisnap-init.sh

```diff
@@ -110,7 +111,7 @@ main() {
     set_custom_tags
 
     # shellcheck disable=SC2086
-    exec ${SWISNAP_HOME}/sbin/swisnapd --config ${CONFIG_FILE} ${FLAGS}
+    exec ${SWISNAP_HOME}/sbin/swisnapd --config ${CONFIG_FILE} ${FLAGS} --log-path ""
```

Comment:
@costap - this is sth you'll like :)

Reply:
also, there's a JIRA task for making it generally available through regular snap config: https://swicloud.atlassian.net/browse/AO-13921

Reply:
@costap this line https://github.com/solarwinds/solarwinds-snap-agent-docker/pull/13/files/dc4dac1650a947158ffe60cc0a304074ad226c79#diff-fcd5e5ce74268c15f79d62441278ce7cR19 fixed the probe failures as well.